### DISCUSSION TITLE
Skywalking installation

### DISCUSSION THREAD

**Discussion by krishnaprakashthokala** [apache/skywalking#12024] [Q&A]
Hi , I reinstalled again used below command and now i see 2 errors in pods. elasitc search has readinessprobe issue.
but how can i change initialdelayseconds to that because i didnt see Replicaset for elastic search . And one more issue 
helm ui pod, 

helm install "${SKYWALKING_RELEASE_NAME}" \
  oci://registry-1.docker.io/apache/skywalking-helm \
  --version "${SKYWALKING_RELEASE_VERSION}" \
  -n "${SKYWALKING_RELEASE_NAMESPACE}" \
  --set oap.image.tag=9.6.0 \
--set oap.storageType=elasticsearch \
  --set ui.image.tag=9.6.0

<img width="944" alt="errorskywalking" src="https://github.com/apache/skywalking/assets/71684984/8dce1644-9aa0-49a1-b08a-cb6b47680eef">
![image](https://github.com/apache/skywalking/assets/71684984/edf4f09c-bafb-44a7-afb2-c687f308db77)

  
![image](https://github.com/apache/skywalking/assets/71684984/4559ac5f-0b2d-4ca0-a354-edd32f3e7b6b)

_Originally posted by @krishnaprakashthokala in https://github.com/apache/skywalking/discussions/12022_

**Comment by wu-sheng** [comment]
Stop doing this. You are repeating creating this.
If you keep doing this, I have to ask the ASF infra to ban you.
